.github/actions/spell-check/whitelist/whitelist.txt

```diff
@@ -1555,12 +1517,10 @@ NEWTEXTMETRIC
 NEWTEXTMETRICEX
 Newtonsoft
 NEXTLINE
-nf
 nfe
 nlength
 Nls
 NLSMODE
-nn
 nnn
```

Comment:
wait why is `nnn` still needed?

Reply:
Looks like it's this:
https://github.com/microsoft/terminal/blob/a13ccfd0f5c1a63ba602d78afba72ab3abaabf34/src/tools/closetest/closetest.cpp#L36
The `:` doesn't fit with the rule. The rule could probably be adjusted. (Or the text changed.)

Reply:
(Would `###:` be better?)

Reply:
I'm fine with either, honestly :smile: ### might be clearer

Reply:
I went w/ `###` and I've updated check-spelling.